app/lib/helpers/job_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Helpers
+  module JobHelper
+    class << self
+      def correct_feature?(job_invocation, feature)
+        job_invocation.job_category == ::ForemanLeapp::JOB_CATEGORY &&
+          RemoteExecutionFeature.find_by(job_template_id: job_invocation.template_invocations[0]
+                                                                        &.template_id)
```

Comment:
This will cause #39 because template invocations get created later in the execution process. Could you check if using `job_invocation.pattern_template_invocations.first` would help?

Reply:
Yop that helped, PR updated.
BUt #39 is still there, tab is displayed but content is empty.